.github/workflows/nightly.yml

```diff
@@ -2,110 +2,411 @@ name: Nightly build
 
 on:
   schedule:
-    - cron: '0 0 * * *' # run at midnight UTC
+    - cron: "0 0 * * *" # run at midnight UTC
   workflow_dispatch:
 
 jobs:
   build-archetype:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - name: Set up Java SE 8
-        uses: actions/setup-java@v3
-        with:
-          distribution: 'temurin'
-          java-version: 8
 
       - name: Cache Maven packages
         uses: actions/cache@v3
         with:
           path: ~/.m2
-          key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
-          restore-keys: ${{ runner.os }}-m2
+          key: ${ runner.os }-m2-${ hashFiles('**/pom.xml') }
+          restore-keys: ${ runner.os }-m2
 
       - name: Build Archetype
         run: mvn clean install --file archetype/pom.xml
 
-      - name: Run Archetype for EE 8, SE 8
+      - name: Set up Java SE 8
+        uses: actions/setup-java@v3
+        with:
+          distribution: "temurin"
+          java-version: 8
+
+      - name: Run Archetype for EE 9.1 Full Profile, SE 8, None
```

Comment:
Please fully describe each case. Is it using Docker or not? Also, no need to specify "none" - it is the default and should be  the first in terms of order.

Reply:
Will it be alright if the name is set in the following format:

For **None** runtime:
`- name: Run Archetype for EE {jakarta-version} {profile-name} Profile, SE {java-version}, with{out} docker`

For **other** runtimes:
`- name: Run Archetype for EE {jakarta-version} {profile-name} Profile, SE {java-version}, {runtime}, with{out} docker`

Reply:
Yes, except please capitalize Docker.

Reply:
Sure. Thank you for the clarification.